app/javascript/src/components/Dashboard/Contacts/ContactTable.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Avatar, Button, Checkbox, Tooltip } from "neetoui";
+
+export default function ContactTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+  onContactDelete,
+}) {
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
+        <thead>
+          <tr className="text-left">
+            <th className="w:auto text-tableTitle">
```

Comment:
I don't think we need custom classes here.

Reply:
@karthiknmenon, removing this class is center aligning the header columns.

Reply:
@karthiknmenon, it's fixed in #14 . Thanks for the suggestion